cern-notebook/Dockerfile.j2

```diff
@@ -0,0 +1,27 @@
+FROM {{ parent }}
+LABEL MAINTAINER="Sebastian Larsen Prehn <slp@erda.dk>
+ARG PACKAGE_TIMEOUT=60
+
+USER root
+
+# Setup default timeout of installing packages
+RUN echo "[global]" > /etc/pip.conf \
+    && echo "timeout=$PACKAGE_TIMEOUT" >> /etc/pip.conf
+
+WORKDIR /tmp
+
+USER $NB_UID
+
+# Install ROOT
+COPY environment.yml /tmp/
+RUN mamba env update -n python3 -f environment.yml \
+    && mamba clean --all -f -y \
+    && fix-permissions $CONDA_DIR \
+    && fix-permissions /home/$NB_USER
+RUN conda install root -c conda-forge
+
+USER root
```

Comment:
USER root at line 23 is also a bit redundant, since we don't need it to change the WORKDIR